nixos/modules/services/web-servers/traefik.nix

```
@@ -48,6 +48,11 @@ let
     ''
   else
     cfg.staticConfigFile;
+
+  finalStaticConfigFile =
+    if cfg.environmentFiles == []
+    then staticConfigFile
+    else "/var/run/traefik/config.toml";
```

```suggestion
    else "/etc/traefik/config.toml";
```
or just /run, nixos is not using /var/run

---

Changed to `/run` so secrets from environment variables aren't leaked to `/etc`.